### ISSUE TITLE
How can you do version detection with Yarn?

### ISSUE THREAD

**Issue by tamj0rd2** [giggio/node-chromedriver#311]
We use yarn for our dependency management. The solutions provided in the docs don't seem to work when used with yarn.

**Comment by giggio** [comment]
Solution for what? Chrome version detection?

**Comment by tamj0rd2** [comment]
Yes sorry, these options don't seem to work when I've tried them replacing `npm` with `yarn`:

```
npm install chromedriver --chromedriver_version=LATEST
CHROMEDRIVER_VERSION=LATEST npm install chromedriver
CHROMEDRIVER_VERSION=LATEST_80 npm install chromedriver
```

**Comment by giggio** [comment]
I just tested with:

````bash
CHROMEDRIVER_VERSION=LATEST_80 yarn add chromedriver
````

And it worked perfectly, version 80 was installed.

Tested with Yarn version `1.22.10`.
Can you send repro steps?

**Comment by tamj0rd2** [comment]
I'm not sure why I had this issue as I'm not able to reproduce it now. It works fine the way you've said. Thanks!

**Comment by tamj0rd2** [comment]
Just as I've said that I've been able to reproduce the issue. I have a `.yarnrc` file with this line in it: `detect_chromedriver_version=true`

After a year install, I start to see errors when I execute this code:

```typescript
const driver = await new Builder()
    .forBrowser('chrome')
    .setChromeOptions(
      new chrome.Options()
        .headless()
        .addArguments('--no-sandbox', '--disable-setuid-sandbox', '--disable-dev-shm-usage')
        .windowSize({ width: 1600, height: 800 })
    )
    .build()
```

The same happens when I use `DETECT_CHROMEDRIVER_VERSION=true yarn install`

**Comment by giggio** [comment]
It'll not work with `yarn install`, only with `yarn add`. The same thing happens with npm, if you run `CHROMEDRIVER_VERSION=LATEST_80 npm i` with a `package.json` which already has `chromedriver` in its dependencies it'll not version 80, but the latest. It will work with `CHROMEDRIVER_VERSION=LATEST_80 npm i chromedriver`.

**Comment by tamj0rd2** [comment]
Thanks. Using yarn add instead of yarn install works like a charm
